### PR TITLE
85 add the mouse compatibility to the menu

### PR DESCRIPTION
**Pull request by Quentintnrl**

### CODE REVIEW

File: core/menu/Menu.cpp [hunk 1]
```diff
@@ -382,3 +385,41 @@ std::string Arcade::Menu::getPlayerName() const
...
+
+void Arcade::Menu::catchMousePosition(int x, int y)
+{
+    _mousePos.first = x;
+    _mousePos.second = y;
+}
+
+void Arcade::Menu::manageClick()
+{
+    std::cout << "Mouse clicked at " << _mousePos.first << " " << _mousePos.second << std::endl;
```
Comment (Njord201) [inline core/menu/Menu.cpp]: remove debug

File: src/sdl/sdl.hpp
Comment (Njord201) [inline src/sdl/sdl.hpp:1]: Pareil avec @6im0n

**Review by Njord201 — APPROVED** [review]

File: src/sdl/sdl.cpp
Comment (6im0n) [inline src/sdl/sdl.cpp:1]: Duplicate, this this part of an issue #56  not here  (Ok for your local test)

you can leave the mouse event

File: src/ncurses/ncurses.hpp
Comment (6im0n) [inline src/ncurses/ncurses.hpp:1]: Ok for mouse event

File: src/sdl/sdl.hpp
Comment (6im0n) [inline src/sdl/sdl.hpp:1]: Same as the .ccp file for the SDL

**Review by 6im0n — CHANGES_REQUESTED** [review]
Please review that change, it will be good !

**Review by 6im0n — APPROVED** [review]
Good

**Pull request merged**